Without this, data get installed into share/cmake-3.22, while the host cmake
(built by cmake-host) at startup expects them to be in share/cmake-3.22.1
and hence fails to detect the cmake root.  From Msys2, mingw-w64-cmake,
0003-Fix-install-destinations.patch.

diff -Nru cmake-3.22.1-orig/Source/CMakeInstallDestinations.cmake cmake-3.22.1-patched/Source/CMakeInstallDestinations.cmake
--- cmake-3.22.1-orig/Source/CMakeInstallDestinations.cmake	2021-12-07 10:44:21.000000000 -0500
+++ cmake-3.22.1-patched/Source/CMakeInstallDestinations.cmake	2022-03-23 15:32:47.110924489 -0400
@@ -6,7 +6,7 @@
   set(CMAKE_INFO_DIR_DEFAULT "documentation/info") # HAIKU
   set(CMAKE_MAN_DIR_DEFAULT "documentation/man") # HAIKU
   set(CMAKE_XDGDATA_DIR_DEFAULT "share") # HAIKU
-elseif(CYGWIN)
+elseif(CYGWIN OR MINGW)
   set(CMAKE_BIN_DIR_DEFAULT "bin") # CYGWIN
   set(CMAKE_DATA_DIR_DEFAULT "share/cmake-${CMake_VERSION}") # CYGWIN
   set(CMAKE_DOC_DIR_DEFAULT "share/doc/cmake-${CMake_VERSION}") # CYGWIN
